### PR TITLE
Update parameters.cljs example to quote query before being passed to parser

### PR DESCRIPTION
**Pull request by tylernisonoff**
```
(clojure.core.async/<!! (parser {} [(::instruments {:sort :instrument/brand})])))
```
will evaluate the sub-expression ` [(::instruments {:sort :instrument/brand})]` before it reaches the parser function. Thus, if we want to specify a parameter, we have to make sure it's quoted.

I also added a Note about this in the docs, but I wasn't sure if this was wanted. Happy to remove if it's not warranted.

### CODE REVIEW

File: docs-src/modules/ROOT/pages/connect/resolvers.adoc [hunk 1]
```diff
@@ -452,6 +452,22 @@ Try it out:
 <div class="space"></div>
 ++++
 
+
+*Note:* If you are calling the parser directly, be sure to quote your query when using parameters like so:
+```clojure
+(parser {} (parser {} '[(::instruments {:sort :instrument/brand})])
```
Comment (souenzzo) [inline docs-src/modules/ROOT/pages/connect/resolvers.adoc:458]: you repeated the parser

File: docs/index.adoc [hunk 2]
```diff
@@ -673,6 +673,21 @@ Try it out:
 <div class="space"></div>
 ++++
 
+*Note:* If you are calling the parser directly, be sure to quote your query when using parameters like so:
+```clojure
+(parser {} (parser {} '[(::instruments {:sort :instrument/brand})])
```
Comment (souenzzo) [inline docs/index.adoc:678]: here too

File: docs-src/modules/ROOT/pages/connect/resolvers.adoc (hunk 1 above)
Comment (tylernisonoff) [reply docs-src/modules/ROOT/pages/connect/resolvers.adoc:458]: ahh sorry about that! not sure what happened there..

**Comment by tylernisonoff** [comment]
Thanks for the comments, updated

**Comment by wilkerlucio** [comment]
Thanks!

**Pull request merged**